instantwm.c

```diff
@@ -435,12 +435,25 @@ void checkanimate(Client *c, int x, int y, int w, int h, int frames, int resetpo
 // move client to position within a set amount of frames
 void animateclient(Client *c, int x, int y, int w, int h, int frames, int resetpos)
 {
-	int time;
-	int oldx, oldy;
 	int width, height;
 	width = w ? w : c->w;
 	height = h ? h : c->h;
 
+	// halve frames if enough events are queried
+	frames = frames / 1 + (XEventsQueued(dpy, QueuedAlready) < 50);
+
+	// No animation if even more events are queried
+    if (!frames || XEventsQueued(dpy, QueuedAlready) > 100) {
+        if (resetpos)
+            resize(c, c->x, c->y, width, height, 0);
+        else
+            resize(c, x, y, width, height, 1);
+        return;
```

Comment:
You should indent this part with tab character

Reply:
No, we should use an autoformatter in our CI so to avoid any arguments over whitespaces and formatting.